modules/gltf/src/lib/extensions/KHR_draco_mesh_compression.js

```diff
@@ -9,7 +9,7 @@ import {getGLTFAccessors, getGLTFAccessor} from '../gltf-utils/gltf-attribute-ut
 
 // Note: We have a "soft dependency" on Draco to avoid bundling it when not needed
 export async function decode(gltfData, options, context) {
-  if (!options.gltf.decompressMeshes) {
+  if (!options.gltf.decompress) {
```

Comment:
I intentionally changed this to `decompressMeshes` but we can still check for the old setting. 

Though I'd prefer you changed in the call site.

There is a function in GLTFLoader that converts old style options to new style.